app/assets/stylesheets/forever_style_guide/modules/_list.scss

```diff
@@ -70,3 +99,18 @@
   list-style: lower-roman;
   list-style-position: outside;
 }
+
+.panel {
+  &.panel-settings {
+    border: 0;
+
+    > .panel-heading {
```

Comment:
We typically don't use these type of selectors in the style guide since it creates a markup dependency. But I suspect this is overrriding a Bootstrap default, in which case it's totally fine, but you may want to add a comment as such.